apps/smart-contracts/core/test/unit/PrePOMarket.test.ts

```diff
@@ -705,20 +714,35 @@ describe('=> prePOMarket', () => {
       const shortToRedeem = amountMinted
       const totalOwed = await calculateTotalOwed(longToRedeem, shortToRedeem, false)
 
-      await prePOMarket.connect(user).redeem(longToRedeem, shortToRedeem)
+      await prePOMarket.connect(user).redeem(longToRedeem, shortToRedeem, user.address)
 
       expect(await longToken.balanceOf(user.address)).to.eq(amountMinted.sub(longToRedeem))
       expect(await shortToken.balanceOf(user.address)).to.eq(amountMinted.sub(shortToRedeem))
       expect(await collateralToken.balanceOf(user.address)).to.eq(totalOwed)
     })
 
+    it('correctly settles redemption with recipient != redeemer before market end', async () => {
+      expect(user.address).not.eq(recipient.address)
+
+      const amountMinted = await setupMarket()
+      const longToRedeem = amountMinted
+      const shortToRedeem = amountMinted
+      const totalOwed = await calculateTotalOwed(longToRedeem, shortToRedeem, false)
```

Comment:
the 'Arrange' section of test cases should always be one group
```suggestion
      expect(user.address).not.eq(recipient.address)
      const amountMinted = await setupMarket()
      const longToRedeem = amountMinted
      const shortToRedeem = amountMinted
      const totalOwed = await calculateTotalOwed(longToRedeem, shortToRedeem, false)
```